scaleway/helpers_lb.go

```diff
@@ -293,3 +293,33 @@ func lbUpgradeV1RegionalToZonedID(element string) (string, error) {
 	//  append zone 1 as default: e.g. fr-par-1
 	return fmt.Sprintf("%s-1/%s", locality, id), nil
 }
+
+func expandLbPrivateNetworkStaticConfig(raw interface{}) *lb.PrivateNetworkStaticConfig {
+	if raw == nil || len(raw.([]interface{})) < 1 {
+		return nil
+	}
+	return &lb.PrivateNetworkStaticConfig{
+		IPAddress: expandStrings(raw),
+	}
+}
+
+func flattenLbPrivateNetworkStaticConfig(cfg *lb.PrivateNetworkStaticConfig) interface{} {
+	if cfg == nil {
+		return nil
+	}
+	return cfg.IPAddress
+}
+
+func expandLbPrivateNetworkDHCPConfig(raw interface{}) *lb.PrivateNetworkDHCPConfig {
+	if raw == nil || len(raw.([]interface{})) < 1 {
+		return nil
+	}
+	return &lb.PrivateNetworkDHCPConfig{}
```

Comment:
Is it normal that the returned value does not depend on raw?

Reply:
Yes, the information is just `PrivateNetworkDHCPConfig{}` to tell to use DHCP, no more data is present in this struct.